examples/guestbook/README.md

```diff
@@ -83,15 +158,17 @@ kind: ReplicationController
 metadata:
   name: redis-master
   labels:
-    name: redis-master
+    app: redis
+    tier: backend
+    role: master
 spec:
   replicas: 1
-  selector:
```

Comment:
We could also leave a comment in place about selector defaulting, in case the user is wondering why this is different than services.